sdk/spring/azure-spring-boot-samples/azure-spring-data-sample-gremlin/README.md

```diff
@@ -1,112 +1,4 @@
-
-## Update code according to this sample project
-
-1. Add dependency of `azure-spring-data-gremlin`, just like the repo does. Choose the correct version.
-
-1. Delete all contents in `src/test/`
-
-1. Add all java files in `src/main/java`, just like this sample does.
-
-1. Update config in `src/main/resorces/application.properties`, where:
-
-   | Field              | Description                                                                                                                                                                                                             |
-   |--------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-   | `endpoint`         | Specifies the Gremlin URI for your database, which is derived from the unique **ID** that you specified when you created your Azure Cosmos DB earlier in this tutorial.                                                 |
-   | `port`             | Specifies the TCP/IP port, which should be **443** for HTTPS.                                                                                                                                                           |
-   | `username`         | Specifies the unique **Database id** and **Graph id** that you used when you added your graph earlier in this tutorial; this must be entered using the following syntax: "/dbs/**{Database id}**/colls/**{Graph id}**". |
-   | `password`         | Specifies either the primary or secondary **Access key** that you copied earlier in this tutorial.                                                                                                                      |
-   | `sslEnabled`       | Specifies whether enable ssl.                                                                                                                                                                                           |
-   | `telemetryAllowed` | Specify **true** if you want to enable telemetry; otherwise, **false**.
-   | `maxContentLength` | Specifies max content length.                                                                                                                                                                                           |
-
-1. About how to get password:
-
-   ![get-password](readme-images/get-password.png)
-
-## Build and run the project
-
-1. Build your Spring Boot application with Maven and run it; for example:
-
-   ```shell
-   mvn clean package
-   mvn spring-boot:run
-   ```
-
-1. If your app start successfully, you can check the graph in Azure portal:
-
-   ![execute-result](readme-images/execute-result.png)
+[spring data gremlin document]: https://docs.microsoft.com/en-us/azure/developer/java/spring-framework/configure-spring-data-gremlin-java-app-with-cosmos-db
```

Comment:
We should remove the en-us part from the URL.

Reply:
Done.